File: cs50x/pset6/sentimental-cash/cash.py
from cs50 import get_float


def main():

    while True:
        cash = get_float("Change: ")
        if cash <= 0:
            continue
        break
    change = 0
    while (cash > 0):
        if (cash >= 0.25):
            cash -= 0.25
        elif (cash >= 0.1):
            cash -= 0.1
        elif (cash >= 0.05):
            cash -= 0.05
        else:
            cash -= 0.01
        cash = round(cash, 2)
        change += 1
    print(f"{change}")


main()

File: cs50x/pset1/cash/cash.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int change, coinCtr = 0;
    do
    {
        change = get_int("Change owed: ");
    }
    while (change < 0);

    while (change > 0)
    {
        if (change >= 25)
        {
            change -= 25;
        }
        else if (change >= 10)
        {
            change -= 10;
        }
        else if (change >= 5)
        {
            change -= 5;
        }
        else
        {
            change--;
        }
        coinCtr++;
    }
    printf("%i\n", coinCtr);
}
